src/DatumEither.ts

```diff
@@ -38,11 +38,21 @@ import { pipe, pipeable } from 'fp-ts/es6/pipeable';
 import * as D from './Datum';
 import { Option } from 'fp-ts/es6/Option';
 import { Lazy, constant, FunctionN, flow } from 'fp-ts/es6/function';
-import { sequenceS, sequenceT } from 'fp-ts/es6/Apply';
-import { Applicative } from 'fp-ts/es6/Applicative';
+import { Apply2, sequenceS, sequenceT } from 'fp-ts/es6/Apply';
+import { Applicative2 } from 'fp-ts/es6/Applicative';
+import * as applicative from 'fp-ts/es6/Applicative';
```

Comment:
Is there a reason for pulling in the entire Applicative module?

Reply:
I think was an accident actually. I thought I was avoiding a name clash but that doesn't appear to be the case.

FWIW, reading some comments on the fp-ts issues gave me the impression that qualified imports like this can still "tree-shake" all the unused functions. This may be incorrect, and either way this change isn't needed so I'll undo this.